src/main/java/util/Utility.java

```diff
@@ -72,7 +72,10 @@ public static boolean writeFile(String fileName, String content, int issueCount)
                 writer.println(content);
                 writer.close();
 
-                long sizeAfterWrite = Files.size(Paths.get(fileName));
+                long sizeAfterWrite = 0;
+                if (issueCount > 0) {
```

Comment:
I am not sure if I understood you correctly, but the check is necessary because `Files.size` would throw an `IOException` if `issueCount` is 0.

Since we should account for 0 `issueCount`, I put it such that no exception would be thrown in such case. An alternative would be handling 0 `issueCount` in the `catch` such that the return value of `writeFile` is consistent (`false` when no error including 0 `issueCount`, `true` when error).